charts/templates/statefulset.yaml

```diff
@@ -206,6 +206,12 @@ spec:
                             sleep 10s
                         fi
                     done
+          {{- if .Values.containerSecurityContext.enabled }}
+          securityContext:
+            runAsUser: {{ .Values.containerSecurityContext.runAsUser }}
```

Comment:
After a second thought, we should still support toYaml to put all the settings from values file so that user and provide customized settings other than the one we provided. 
To get rid of the enabled entry, we can use this: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}  
I've also updated the function spec.

Reply:
updated.